### PR TITLE
specs actors update

### PR DESCRIPTION
**Pull request by arajasek**
Needs
- [x] [chain-val](https://github.com/filecoin-project/chain-validation/pull/215) (let's actually merge the PR this time)
- [x] [storage-fsm](https://github.com/filecoin-project/storage-fsm/pull/45) (somewhat dubious change for tests)

### CODE REVIEW

**Comment by magik6k** [comment]
Probably shouldn't merge before actors land a tag

**Comment by arajasek** [comment]
Updated to use the v0.7.2 tag, still needs testfixing

File: chain/gen/genesis/miners.go [hunk 1]
```diff
@@ -271,6 +271,8 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sroot cid.Cid
 		return cid.Undef, xerrors.Errorf("mutating state: %w", err)
 	}
 
+	// TODO: Should we re-ConstructState for the reward actor using rawPow as currRealizedPower here?
```
Comment (magik6k) [inline chain/gen/genesis/miners.go:274]: Good question, I feel like it is set correctly when we import sectors, but that warrants checking, will open an issue

Comment (magik6k) [reply chain/gen/genesis/miners.go:274]: Opened https://github.com/filecoin-project/lotus/issues/2389

File: chain/vm/runtime.go [hunk 2]
```diff
@@ -578,3 +578,16 @@ func (rt *Runtime) abortIfAlreadyValidated() {
 	}
 	rt.callerValidated = true
 }
+
+func (rt *Runtime) Log(level vmr.LogLevel, msg string, args ...interface{}) {
+	switch level {
+	case vmr.DEBUG:
+		log.Debugf(msg, args)
```
Comment (magik6k) [inline chain/vm/runtime.go:585]: Should be a separate logger

**Pull request merged**